client/src/components/QuestionsAnswers/QuestionsAnswers.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import API_KEY from '/config.js';
```

Comment:
It doesn't appear you're doing any axios requests in this component, so I don't think you need to import it.

Reply:
Yup, you're correct! Removed the axios and API_KEY import lines, as well as "useEffect."